src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/validator.html.twig

```diff
@@ -0,0 +1,98 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% if collector.data.nb_violations > 0 or collector.data.calls|length %}
+        {% set status_color = collector.data.nb_violations ? 'red' : '' %}
+        {% set icon %}
+            {{ include('@WebProfiler/Icon/validator.svg') }}
+            <span class="sf-toolbar-value">
+                {{ collector.data.nb_violations ?: collector.data.calls|length }}
```

Comment:
I agree. I started the template from the Forms collector one. It's a mistake due to fact I overlooked at it in the first place. 😄 (but I wonder if it really make more sense for forms to switch between errors and forms count)
We should always show the number of violations instead.